### PR TITLE
Add docker options to scheduler API docs

### PR DESCRIPTION
**Pull request by solnic**

### CODE REVIEW

File: scheduler/docs/configuration.asc [hunk 1]
```diff
@@ -98,6 +98,39 @@ We'll look at the configurable options in turn:
   This sets the role that Cook will connect to Mesos with. Default: `*`
   You can omit this property unless you've enabled security features in Mesos. The value should be in authorized list for the current `:principal` in `register_frameworks` Action in Mesos Authorization file. See http://mesos.apache.org/documentation/latest/authorization/ for details.
 
+`:container`::
```
Comment (wyegelwel) [inline scheduler/docs/configuration.asc]: Can you move this to the [scheduler api doc](https://github.com/mojotech/Cook/blob/d356a82bde4b8dc18876db600627b13d0ecf123a/scheduler/docs/scheduler-rest-api.asc#launch-a-job) and follow the same format?

**Comment by solnic** [comment]
@wyegelwel I moved the docs the the correct place :)

**Comment by m4ce** [comment]
```
2016-09-01 10:34:46,909 INFO  cook.mesos.api [qtp675831479-29] - Submitting jobs through raw api: [{:mem 1000.0, :name cookjob, :command echo hello world, :max-runtime 86400000, :po
rts 0, :priority 50, :container {:type DOCKER, :docker {:image centos}}, :max-retries 1, :uuid #uuid "26719da8-394f-44f9-9e6d-8a17500f5110", :user root, :cpus 1.5}]
2016-09-01 10:34:46,911 ERROR cook.mesos.api [qtp675831479-29] - Error submitting jobs through raw api
java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: :db.error/nil-value Nil is not a legal value
```

Any idea why?

**Comment by m4ce** [comment]
Missing network parameter, thanks though.

**Comment by dposada** [comment]
Is this still active or can it be closed?

**Comment by dposada** [comment]
Closing since this seems to be inactive

**Pull request closed without merging**